src/org/usfirst/frc2175/config/IntakeConfig.java

```diff
@@ -0,0 +1,49 @@
+package org.usfirst.frc2175.config;
+
+import java.util.Properties;
+
+public class IntakeConfig extends BaseConfig {
+    private static final String PROPERTY_FILE_NAME = "intake.properties";
+    private double sideBeltSpeedForward;
+    private double sideBeltSpeedReverse;
+    private double mainBeltSpeedForward;
+    private double mainBeltSpeedReverse;
+
+    @Override
+    public String getPropertyFileName() {
+        return PROPERTY_FILE_NAME;
+    }
+
+    @Override
+    protected void configure(Properties properties) {
+        double sideBeltSpeedForward = getIntPropertyValue(
```

Comment:
To clarify...you need to use `getDoublePropertyValue`. Otherwise we'll get errors when we try to load values like 0.5 from the properties file.